modules/customize-certificates-understanding-default-router.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// authentication/certificates/replacing-default-ingress-certificate.adoc
+
+[id="understanding-default-ingress-{context}"]
+= Understanding the default ingress certificate
+
+By default {product-title} uses the Ingress Controller Operator to 
+create an internal CA and issue a wildcard certificate that is valid for 
+applications under the `.apps` sub-domain. Both the web console and CLI
+use this certificate as well. 
+
+The internal infrastructure certificates are self-signed. 
```

Comment:
Only the internal infrastructure *CA* certificates are self-signed.

Reply:
Included this detail.